Modules/Core/Common/include/itkOptimizerParameters.h

```diff
@@ -168,7 +168,8 @@ class ITK_TEMPLATE_EXPORT OptimizerParameters : public Array<TParametersValueTyp
   ~OptimizerParameters() override = default;
 
 private:
-  std::unique_ptr<OptimizerParametersHelperType> m_Helper{ new OptimizerParametersHelperType };
+  std::unique_ptr<OptimizerParametersHelperType> m_Helper =
+    std::unique_ptr<OptimizerParametersHelperType>{ new OptimizerParametersHelperType };
```

Comment:
Note that C++14 introduced `std::make_unique`, which should preferably be used here instead of `new`, once ITK has upgraded to >= C++14.

Reply:
This could be a comment in the source code.

Reply:
@dzenanz Well, I considered adding such a comment, but I guess we might as well just run clang-tidy `modernize-make-unique` over the entire ITK, once ITK is upgraded to >= C++14: https://clang.llvm.org/extra/clang-tidy/checks/modernize-make-unique.html